src/GTP.h

```diff
@@ -69,6 +99,7 @@ extern std::string cfg_options_str;
 extern bool cfg_benchmark;
 extern bool cfg_cpu_only;
 extern int cfg_analyze_interval_centis;
+extern AnalyzeTags cfg_analyze_tags;
```

Comment:
I was thinking where the best place is to store these settings. I ended up storing it here because it is right next to the existing variable `cfg_analyze_interval_centis`.

By adding the new class, I intended to make it straightforward to implement more of these tags in the future.